include/dts-functions.sh

```diff
@@ -1431,6 +1445,10 @@ main_menu_options(){
 
         # Use regular update process for everything else
         ${CMD_DASHARO_DEPLOY} update
+        result=$?
+        if [ "$result" -ne 0 ] && [ "$result" -ne 2 ]; then
+          send_dts_logs ask
```

Comment:
```suggestion
          send_dts_logs ask return 0
```

So not to see `Press Enter to continue.` twice.

Reply:
d394c656fe39